docs/src/views/AutoFocusZone.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react'
+import { Header } from '@stardust-ui/react'
+import DocPage from '../components/DocPage'
+import { code, link } from '../utils/helpers'
+import CodeSnippet from '../components/CodeSnippet'
+
+export default () => (
+  <DocPage title="Auto Focus Zone">
+    <Header as="h2">Content</Header>
+    <ul>
+      <li>{link('Overview', '/auto-focus-zone#overview')}</li>
+      <li>{link('Usage', '/auto-focus-zone#usage')}</li>
+      <li>
+        {link(
+          'Override AutoFocusZone settings',
+          '/auto-focus-zone#override-autofocuszone-settings',
+        )}
+      </li>
+    </ul>
+    <Header as="h2">Overview</Header>
+    <p>
+      AutoFocusZone is used to focus inner element on mount. For example, when we want to focus
```

Comment:
sorry don't fully understand here this part of the sentence "but still without focus trap."
is it somehow connected the using "AutoFocusZone" and "focus trap" ? :)

Reply:
chnged :) thank you